### DISCUSSION TITLE
Release of zlib-ng 2.1.0

### DISCUSSION THREAD

**Discussion by yetanotherspinner** [zlib-ng/zlib-ng#1366] [General]
Hi! Are there any plans to make a stable release of zlib-ng (e.g. 2.1.0) based on the current development version? The last stable release is soon to be 1-year old... AFAIK, some of zlib's known vulnerabilities (which zlib-ng is also affected with) were fixed in zlib-ng's code. Thanks

**Comment by mtl1979** [comment]
This has been discussed before... There is still a few issues that need to be resolved before we make a stable release.

**Comment by Dead2** [comment]
If anyone wants to start building a list of PRs that should get backported to a new 2.0.x stable release, then that would be great. Separated into two or three levels of priority: important fixes, less important fixes and nice to have. No big stuff that can be avoided of course.
I really do want to get a new 2.0 stable release out, it is much overdue 😞.

2.1.0 has more things queued, as well as a breaking ABI change that I want to suggest and hopefully get implemented, one that I wish we had in 2.0 already, but better in 2.1 than wait even more.


I am unfortunately very swamped with a personal project that Really NEEDS to be done before the snow starts coming down.
(Had to tear down the old staircase leading to the house because it was breaking up and quite unsafe due to a lousy foundation, who knew sand on top of clay is not a good foundation.. 🤦 😠. So we have spent weeks digging out tons of clay, laying down insulation etc, putting 6 tons of sand, now we just need to lay down the paving stones and then build a new entrance staircase in wood. Doesn't help that I managed to split one of my fingers quite badly with a knife yesterday, I am lucky the knife stopped when it hit bone. Stupid me didn't think I'd be doing any cutting when I started, so I didn't put on any of the various cut-resistant gloves I have and instead used a cheap pair..  There is already snow in the mountains, so I am really feeling the pressure now..)

**Comment by mtl1979** [comment]
For 2.0.x, we might want to cherry-pick at least commit 9309904fe79cd60599259022bd2aab2a3e1fb0d8 ...

For 2.1.0, we need to fix the ABI check failures. I'm inclined to not merge change from `uint64_t` to `size_t` due to various issues with Visual C++ compiler when using `size_t` and `ssize_t`.
